.github/scripts/run-tests.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Log received parameters
+echo "run-tests.sh received parameters: $@"
+
+# Parse arguments
+while [[ $# -gt 0 ]]; do
+  case $1 in
+    --dtest)
+      DTEST_PATTERN="$2"
+      shift 2
+      ;;
+    *)
+      echo "Unknown option: $1"
+      exit 1
+      ;;
+  esac
+done
```

Comment:
why the need to invent another name ; argument parsing/etc? 
 why not just pass it to the `mvn` comamnd?>

Reply:
Yes, will work on simplifying this script.